scripts/preinstall.js

```diff
@@ -3,10 +3,17 @@ const os = require('os');
 const path = require('path');
 
 const configsDir = path.resolve(os.homedir(), '.bpanel');
+const configsFile = path.resolve(configsDir, './.bpanel.js');
 const clientsDir = path.resolve(configsDir, 'clients');
 
```

Comment:
Maybe the UX would be better if `configText` was a file that users can edit and then you first read the file into a string so that when updating the file, it feels more natural

Reply:
This just for default and is only run once after you first install so that startup doesn't fail when it can't find the config file. Making normal edits happens in a text file though.

Reply:
I'm not sure if we'd want this startup text to be edited anyway to be honest as if it's written incorrectly, the build would fail which wouldn't make for a great ux since it might be hard for a newby to be clear on where that break happened.

Reply:
Ok I didn't realize that was just for the initial startup, I thought that was where the actual config lived

Reply:
👍 Yeah, once this is in, the idea would be to not be maintaining local plugins in the main project repo anymore.